demos/noise_suppression_demo/python/noise_suppression_demo.py

```diff
@@ -30,20 +19,19 @@
 log.basicConfig(format='[ %(levelname)s ] %(message)s', level=log.DEBUG, stream=sys.stdout)
 
 
-def build_argparser():
-    parser = ArgumentParser(add_help=False)
-    args = parser.add_argument_group('Options')
-    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
-    args.add_argument("-m", "--model", help="Required. Path to an .xml file with a trained model",
-                      required=True, type=Path)
-    args.add_argument("-i", "--input", help="Required. Path to a 16kHz wav file with speech+noise",
-                      required=True, type=str)
-    args.add_argument("-o", "--output", help="Optional. Path to output wav file for cleaned speech",
-                      required=False, type=str, default="noise_suppression_demo_out.wav")
-    args.add_argument("-d", "--device",
-                      help="Optional. Target device to perform inference on. "
-                           "Default value is CPU",
-                      default="CPU", type=str)
+def build():
+    parser = ArgumentParser()
+    parser.add_argument("-model", required=True, type=Path, metavar="<[M]odel file>",
+        help="path to an .xml file with a trained [M]odel")
```

Comment:
I agree with @fzhar. Is it possible the underlined letters in brackets to be lowercase, like that: `[m]odel`?   


From the user perspective, I read the help message and try to launch the demo with -M, -I (uppercased) arguments and then get the error "unrecognized arguments". It is not trivial, that it should not work according to the description with uppercase one, so I (as the user) will just use full versions of arguments: -model, -input to avoid the error.

Reply:
C++ and Python should be consistent. There will be no [] then